users/untrusted.rst

```diff
@@ -58,6 +58,10 @@ becomes available.
         T2 -> U1 [label="Encrypted by T2"]
     }
 
+In the above scenarios, if the untrusted device U1 deletes a local copy of a
+file, this is not replicated directly to trusted hosts. Untrusted hosts might
+receive the notice, however.
```

Comment:
I mean, they won't, because Syncthing won't send it. This whole thing is a hypothetical on a potential Evil Syncthing that does the wrong thing but speaks the same protocol. 🤷

Reply:
I understand better now. Just the sentence is a good add.